lib/engines/content_block_manager/app/models/content_block_manager/content_block/document.rb

```diff
@@ -6,7 +6,7 @@ class Document < ApplicationRecord
       include Scopes::SearchableByUpdatedDate
 
       extend FriendlyId
-      friendly_id :title, use: :slugged, slug_column: :content_id_alias, routes: :default
+      friendly_id :sluggable_string, use: :slugged, slug_column: :content_id_alias, routes: :default
```

Comment:
FWIW I think it's fine to not update the slug when we update the title, as they should be immutable. This will be particularly important when we move over to use slugs over the UUIDs in the embed codes, though I _think_ we can have multiple aliases for blocks.

Reply:
do you think the `content_id_alias` could just be assigned directly then, rather than going through the `sluggable_string`? I wasn't sure what was going on with the FriendlyId stuff tbh.

Reply:
I'll leave as it is for now as I don't want to unpick the friendlyID stuff or risk it getting out of sync with how the other aliases are generated elsewhere, but we can always return to it